test/extended/apiserver/resiliency.go

```diff
@@ -0,0 +1,66 @@
+package apiserver
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/onsi/ginkgo"
+	"github.com/onsi/gomega"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/uuid"
+	"k8s.io/apimachinery/pkg/util/wait"
+	clientset "k8s.io/client-go/kubernetes"
+	"k8s.io/kubernetes/test/e2e/framework"
+
+	exutil "github.com/openshift/origin/test/extended/util"
+	"github.com/openshift/origin/test/extended/util/disruption"
+	"github.com/openshift/origin/test/extended/util/disruption/controlplane"
+	types "k8s.io/apimachinery/pkg/types"
+	"k8s.io/kubernetes/test/e2e/upgrades"
+)
+
+func forceApiRollout(oc *exutil.CLI) {
+	redeploymentReason := fmt.Sprintf(`{"spec":{"forceRedeploymentReason":"resiliency-test-%v"}}`, uuid.NewUUID())
+
+	_, err := oc.AdminOperatorClient().OperatorV1().KubeAPIServers().Patch(context.Background(), "cluster", types.MergePatchType,
+		[]byte(redeploymentReason), metav1.PatchOptions{})
+	gomega.Expect(err).NotTo(gomega.HaveOccurred())
+}
+
+func isApiUp(client clientset.Interface) (bool, error) {
+	_, err := client.CoreV1().RESTClient().Get().RequestURI("/readyz").DoRaw(context.TODO())
+	return err == nil, err
+}
+
+var _ = ginkgo.Describe("[Conformance][sig-api-machinery][Serial]", func() {
```

Comment:
This is wrong. `[sig-sno]`. We don't own this.

Reply:
hm... isnt it api behavior?

Reply:
yet, who is getting BZs about SNO problem? My last information was that this is the telco team.

Reply:
Note that I read "Ultimately, it will make validations on other cluster components, but for now we'll validate API is getting up fast enough (40 seconds of outage)." as it will get operator recovery tests as well here.

Reply:
see comment below https://github.com/openshift/origin/pull/26337/files#r686730057

Reply:
sure, but how test coverage related to customer BZ? I mean failure in this test will probably cause API bz, no? At least untill the operators part is introduced

Reply:
@osherdp it can be sig-sno, I dont really mind.